# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xalan-c/hotfix-cxx17.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Build as C++17, required by the ICU headers.

--- xalan_c-1.12.orig/CMakeLists.txt	2026-09-27 18:44:29.844155095 +0200
+++ xalan_c-1.12/CMakeLists.txt	2026-09-27 18:44:29.849477006 +0200
@@ -34,7 +34,7 @@ endif(POLICY CMP0067)
 
 # Try C++14, then fall back to C++11 and C++98.  Used for feature tests
 # for optional features.
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
 
 # Use folders (for IDE project grouping)
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
